### ISSUE TITLE
[Bug]: Patch Operation Failed: GoJuice minToleranceToAddict 

### ISSUE THREAD

**Issue by Lobz** [CombatExtended-Continued/CombatExtended#110]
**Specifications**
Rimworld version: 1.1.2647
Combat Extended version: 1.1.2.1
Combat Extended source (Steam, GitHub, etc.): Steam
Your operating system: Windows 8
Your mod list:
1. Harmony
2. Core
3. Combat Extended 

**Description**
On game opening, debug log shows this error
```
[Combat Extended] Patch operation Verse.PatchOperationReplace(Defs/ThingDef[defName="GoJuice"]/comps/li[@Class="CompProperties_Drug"]/minToleranceToAddict) failed
file: C:\Program Files (x86)\Steam\steamapps\workshop\content\294100\1631756268\Patches\Core\Drugs\Drugs.xml
```

**Expected behavior**
Game opening with no errors

**To reproduce**
Open Rimworld with CE active on the mod list

**Screenshots & log dumps**
[RimWorld1.1.2647rev894.txt](https://github.com/CombatExtended-Continued/CombatExtended/files/4706238/RimWorld1.1.2647rev894.txt)

**Complete the following checklist**
I hereby verify that I have done the following:
- [x] Confirmed that my game version and load order are correct.
- [x] Confirmed that I am running the appropriate and most updated version of Combat Extended and required compatibility patches.
- [x] Confirmed I am not running any mods with known incompatibilities with Combat Extended.
- [x] Disabled Combat Extended and attempted to reproduce the behavior without success.

**Comment by Lobz** [comment]
Ok it seems this issue has been fixed by PR #104 
It need to be updated to Steam.

**Comment by N7Huntsman** [comment]
Fixed in the development build, and will be in the next release.
